zerver/webhooks/uptimerobot/doc.md

```diff
@@ -1,33 +1,47 @@
+# Zulip UptimeRobot integration
+
 Receive Zulip notifications from UptimeRobot!
 
+{start_tabs}
+
 1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
-1. {!generate-integration-url.md!}
+1. {!generate-webhook-url-basic.md!}
 
-1. On UptimeRobot, go to **My Settings** and select **Add Alert Contact**.
-   Set **Alert Contact Type** to **webhook**.
+1. On UptimeRobot, go to **My Settings** and select **Add Alert Contact**.<br>
+   Set **Alert Contact Type** to **webhook**.  
+   Set **Friendly Name** to a name of your choice, such as `Zulip`.  
```

Comment:
Reformatted the instructions in an attempt to:
- group related steps (they could be grouped further too, with the tradeoff of having a single large instruction)
- keep each substep in its own line for readability, but without using an ordered/unordered list.

Reply:
We've been trying to keep to a format where there's:
- A single instruction for navigating
- A single instruction (or grouped by theme if there are a lot) for setting things

Here, I'd go with something like ...

```
1. On UptimeRobot, got to **My Settings**, and select **Add Alert Contact**.

1. Set **Alert Contact Type** to **webhook**, and set **Friendly Name** ...
```